gapis/trace/android/profile/profile.go

```diff
@@ -82,13 +84,13 @@ func setTimeMetrics(groupToSlices map[int32][]*service.ProfilingData_GpuSlices_S
 	*metrics = append(*metrics, &service.ProfilingData_GpuCounters_Metric{
 		Id:   gpuTimeMetricId,
 		Name: "GPU Time",
-		Unit: "ns",
+		Unit: fmt.Sprint(device.GpuCounterDescriptor_MeasureUnit_value[device.GpuCounterDescriptor_NANOSECOND.String()]),
```

Comment:
You don't need to go through all these mappings (i.e. now it maps int->name->int->string). Just
`strconv.Itoa(device.GpuCounterDescriptor_NANOSECOND)` will do :)

Reply:
Yeah the converting is painful! I actually had a try on the `strconv.Itoa(device.GpuCounterDescriptor_NANOSECOND)` but would get an error of `couldn't use type device.GpuCounterDescriptor_MeasureUnit) as type int`, it seems to be not seeing it as a golang enum. But now I add a smaller wrapper of `int` it seems to work!